### PR TITLE
Preparing of v1.8.0 release

### PR DESCRIPTION
**Pull request by ahmarsuhail**
## Description of change

PR for v1.8.0 release

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and I agree to the terms of the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).

### CODE REVIEW

File: mountpoint-s3/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## v1.8.0
```
Comment (dannycjones) [inline mountpoint-s3/CHANGELOG.md:1]: We should include a note on the bug fixes.

Add something like this after the new features section.

```markdown
### Other changes

* Support added for providing `external_id` when defining AWS profiles in a config file. ([#962](https://github.com/awslabs/mountpoint-s3/pull/962))
* Fix issue where empty environment variables for STS web identity credentials could cause segmentation fault. ([#246](https://github.com/awslabs/aws-c-auth/pull/246))

**Review by dannycjones — CHANGES_REQUESTED** [review]

File: mountpoint-s3/CHANGELOG.md (hunk 1 above)
Comment (ahmarsuhail) [reply mountpoint-s3/CHANGELOG.md:1]: done

File: mountpoint-s3/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
 ### New features
 
 * Mountpoint now offers two new command-line arguments `--read-part-size <SIZE>` and `--write-part-size <SIZE>` which allow to specify different part sizes to be used when reading and writing respectively.
```
Comment (monthonk) [inline mountpoint-s3/CHANGELOG.md:5]: We forgot to add the link to the PR here
```suggestion
* Mountpoint now offers two new command-line arguments `--read-part-size <SIZE>` and `--write-part-size <SIZE>` which allow to specify different part sizes to be used when reading and writing respectively. ([#949](https://github.com/awslabs/mountpoint-s3/pull/949))
```

Comment (dannycjones) [reply mountpoint-s3/CHANGELOG.md:5]: My bad :)

Comment (monthonk) [reply mountpoint-s3/CHANGELOG.md:5]: Not really, I reviewed that PR and just realised we should add it :(

Comment (ahmarsuhail) [reply mountpoint-s3/CHANGELOG.md]: done

**Review by dannycjones — APPROVED** [review]
LGTM! Sorry for the churn

**Review by monthonk — APPROVED** [review]
LGTM

**Pull request merged**